### PR TITLE
Fix/integration environment post cleanup

### PR DESCRIPTION
**Pull request by AshtonStephens**
## Description

Closes: #236, #505, 
Extends: #680 

## Changes

Fixes some bugs in the original devenv env cleanup ticket. 

## Testing Information

Tested with integration tests and locally running the developer environment.

## Checklist:

- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docker/sbtc/signer/signer-config.toml [hunk 1]
```diff
@@ -130,19 +130,31 @@ deployer = "ST2SBXRBJJTH7GV5J93HJ62W2NRRQ46XYBK92Y039"
 # Environment: SIGNER_SIGNER__DB_ENDPOINT
 db_endpoint = ""
 
+# The public keys of the signer sit during the bootstrapping phase of
+# the signers.
+#
+# Required: true
+bootstrap_signing_set = ["035249137286c077ccee65ecc43e724b9b9e5a588e3d7f51e3b62f9624c2a49e46", "031a4d9f4903da97498945a4e01a5023a1d53bc96ad670bfe03adf8a06c52e6380", "02007311430123d4cad97f4f7e86e023b28143130a18099ecf094d36fef0f6135c"]
+
```
Comment (djordon) [inline docker/sbtc/signer/signer-config.toml:138]: Would you mind formatting these like:
```toml
bootstrap_signing_set = [
    "035249137286c077ccee65ecc43e724b9b9e5a588e3d7f51e3b62f9624c2a49e46",
    "031a4d9f4903da97498945a4e01a5023a1d53bc96ad670bfe03adf8a06c52e6380",
    "02007311430123d4cad97f4f7e86e023b28143130a18099ecf094d36fef0f6135c"
]
```

File: docker/sbtc/signer/signer-config.toml [hunk 2]
```diff
@@ -130,19 +130,31 @@ deployer = "ST2SBXRBJJTH7GV5J93HJ62W2NRRQ46XYBK92Y039"
...
+# The number of signatures required for the signers' bootstrapped
+# multi-sig wallet on Stacks.
+#
+# Required: true
+bootstrap_signatures_required = 3
```
Comment (djordon) [inline docker/sbtc/signer/signer-config.toml:143]: The value here should be `2`

**Review by djordon — COMMENTED** [review]
So far so good. I got tired of waiting for the build to finish but I'll come back to this later when I need to relax a little and have more patience for waiting for the build.

**Comment by AshtonStephens** [comment]
It shouldn't take more than 15 minutes, and subsequent builds should take 2 min fwiw

File: docker/sbtc/signer/signer-config.toml (hunk 1 above)
Comment (AshtonStephens) [reply docker/sbtc/signer/signer-config.toml]: Done.

File: docker/sbtc/signer/signer-config.toml (hunk 2 above)
Comment (AshtonStephens) [reply docker/sbtc/signer/signer-config.toml]: Done.

**Review by aldur — APPROVED** [review]

**Pull request merged**